projects/ad9361/src/main.c

```diff
@@ -780,33 +773,10 @@ int main(void)
 	 */
 	struct irq_ctrl_desc *irq_desc;
 
-	/**
-	 * Xilinx platform dependent initialization for UART.
-	 */
-	struct xil_uart_init_param xil_uart_init_par;
-
-	/**
-	 * Initialization for UART.
-	 */
-	struct uart_init_param uart_init_par;
-	struct uart_desc *uart_desc;
-
 	status = irq_ctrl_init(&irq_desc, &irq_init_param);
```

Comment:
I think this is done in the iio_app already:
https://github.com/analogdevicesinc/no-OS/blob/73a13b94aad63e2050270e0ee9287e6e28dc9756/iio/iio_app/iio_app.c#L143-L145